index.ts

```diff
@@ -7,6 +7,7 @@ import { VFileWithOutput } from "unified";
 export interface Heading {
   depth: number;
   value: string;
+  id?: string;
```

Comment:
Does it maybe make sense to pass along whatever data might exist from other plugins? It's been a while since I dug into the remark ecosystem, but if we can add a general case solution here that'd be cool.

```suggestion
  data?: any;
```

Reply:
Great idea!